src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -116,7 +120,9 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
             .get<ModelResultResponse>(`model/comparison/plot/${calibrateId}`);
 
         if (response) {
-            if (response.data){
+            const comparisonPlotSelections = context.rootState.plottingSelections.comparisonPlot
```

Comment:
Again, this is not going to help with the requirements of the ticket, and we shouldn't do this as new result set may have entirely different selections.